lib/google_drive/acl.rb

```diff
@@ -27,7 +27,9 @@ def initialize(session, file)
         api_permissions.permissions.map { |perm| AclEntry.new(perm, self) }
     end
 
-    def_delegators(:@entries, :size, :[], :each)
+    def_delegators(:@entries, :size, :[], :each, :map, :any?, :sample, :select,
```

Comment:
Thanks! But can you let this class include Enumerable instead? It would add most of the methods here and even more.